doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -415,6 +415,56 @@ If your model is sensitive to shuffle quality, call :meth:`Dataset.random_shuffl
 
 For more information on how to optimize shuffling, and which approach to choose, see the :ref:`Optimize shuffling guide <optimizing_shuffles>`.
 
+Preprocessing Data
+------------------
+
+The recommended way to preprocess data for training is to use Ray Data operations such as `map_batches`. 
+See the :ref:`Ray Data Working with Pytorch guide <working_with_pytorch>` for more details.
+
+However, particularly for tabular data, you can also use Ray Data :ref:`preprocessors <air-preprocessors>`, which implement common data preprocessing operations.
+You can use this with Ray Train Trainers by applying them on the dataset before passing the dataset into a Trainer. For example:
+
+.. testcode::
+
+    import numpy as np
+
+    import ray
+    from ray.train import ScalingConfig
+    from ray.train.torch import TorchTrainer
+    from ray.data.preprocessors import Concatenator, Chain, StandardScaler
```

Comment:
For the sake of deduplication, would prefer to do one of the following:

1. Remove this code snippet and just have the user read the `Preprocessors` page.
2. Simplify the `Preprocessors` page to not go in depth into the Ray Train part, and only cover the Ray Train + Ray Data integration in this page.

cc @amogkam

Reply:
Will do 2